README.md

```diff
@@ -129,6 +129,20 @@ that `issx` will interact with.
 an instance.
 `project` field should contain the project id available in the chosen instance (usually it is a number).
 
+### Basic auth
+
+For Redmine instances that use basic authentication, you can specify the credentials using `basic_auth_user` and
+`basic_auth_password` properties in the instance configuration:
+
```

Comment:
I would prefer keeping description of this section generic, and explicitely enumerating supported `instances`.
This way it will be easy to add support for basic auth to the rest of `instances` in the future? What do you think?